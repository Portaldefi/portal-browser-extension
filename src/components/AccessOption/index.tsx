import React from 'react';
import { Checkbox, Grid, Header, Radio } from 'semantic-ui-react';

interface IConnectionItem {
  name: string,
  comment: string
};

export default ({name, comment}: IConnectionItem) => {
  return (
    <Grid className='p-16 py-5 bg-white'>
      <Grid.Column width='eleven' textAlign='left'>
        <p className='font-family-Inter font-weight-400 font-size-14 line-height-17'>{name}</p>
      </Grid.Column>
      <Grid.Column width='five' verticalAlign='middle'>
        <Radio />
      </Grid.Column>
    </Grid>
  )
}