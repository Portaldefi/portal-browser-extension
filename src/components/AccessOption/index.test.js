import AccessOption from './index';
import { create, act } from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Button } from 'semantic-ui-react';

describe('AccessOptionComponent', () => {
  it('should work', () => {
    let tree;
    act(() => {
      tree = create(
        <BrowserRouter>
          <AccessOption />
        </BrowserRouter>
      );
    });

    expect(tree).toMatchSnapshot();
  });
  afterAll(() => jest.resetModules());
});