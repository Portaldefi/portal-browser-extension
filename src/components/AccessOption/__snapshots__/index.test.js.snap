// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccessOptionComponent should work 1`] = `
<div
  className="ui grid p-16 py-5 bg-white"
>
  <div
    className="left aligned eleven wide column"
  >
    <p
      className="font-family-Inter font-weight-400 font-size-14 line-height-17"
    />
  </div>
  <div
    className="middle aligned five wide column"
  >
    <div
      className="ui fitted radio checkbox"
      onChange={[Function]}
      onClick={[Function]}
      onMouseDown={[Function]}
      onMouseUp={[Function]}
    >
      <input
        checked={false}
        className="hidden"
        readOnly={true}
        tabIndex={0}
        type="radio"
      />
      <label />
    </div>
  </div>
</div>
`;
